website/client/js/controllers/inventoryCtrl.js

```diff
@@ -54,11 +54,16 @@ habitrpg.controller("InventoryCtrl",
 
     $scope.$watch('user.items.gear', function(gear){
       $scope.gear = {};
+      $scope.gearByType = {};
       _.each(gear.owned, function(v,key){
         if (v === false) return;
         var item = Content.gear.flat[key];
-        if (!$scope.gear[item.klass]) $scope.gear[item.klass] = [];
+        if (!$scope.gear[item.klass])
```

Comment:
Oh I see!  In my opinion, having everything on one line makes the scope obvious enough that brackets aren't needed.  However, moving it to a new line makes the scope of the `if` a lot less clear, so there should probably be brackets.  Better safe than sorry. :+1: